src/server/scripts/EasternKingdoms/ScarletMonastery/boss_mograine_and_whitemane.cpp

```diff
@@ -295,6 +303,26 @@ class boss_high_inquisitor_whitemane : public CreatureScript
                 damage = me->GetHealth() - 1;
         }
 
+        void SetGUID(ObjectGuid attacker, int32 action) override
```

Comment:
```
src/server/scripts/EasternKingdoms/ScarletMonastery/boss_mograine_and_whitemane.cpp:306:49:
 fatal error: unused parameter 'action' [-Wunused-parameter]
        void SetGUID(ObjectGuid attacker, int32 action) override
                                                ^
```
Comment out the unused parameter ( `int32 /*action*/` ) -- unless you plan to modify the code to use it.

Reply:
Didn't actually mean to push this, it got pushed automatically when I pushed another branch :D gonna take this out.